### PR TITLE
[SDFAB-1178] Allows to read ingress and egress counters separately

### PR DESCRIPTION
**Pull request by daniele-moro**
Depends on:
- https://github.com/stratum/fabric-tna/pull/520
- https://gerrit.onosproject.org/c/onos/+/25447

This PR consider only the read path. Code is taken partially from:
- https://github.com/omec-project/up4/pull/282
- https://github.com/omec-project/up4/pull/279

### CODE REVIEW

File: app/app/src/test/java/org/omecproject/up4/impl/Up4TranslatorImplTest.java [hunk 1]
```diff
@@ -40,6 +41,9 @@ public UpfEntity up4ToUpfEntity(UpfEntity expected, PiEntity up4Entry) {
                 case METER_CELL_CONFIG:
                     translatedEntity = up4Translator.up4MeterEntryToUpfEntity((PiMeterCellConfig) up4Entry);
                     break;
+                case COUNTER_CELL:
+                    translatedEntity = up4Translator.up4CounterEntryToUpfEntity((PiCounterCell) up4Entry);
+                    break;
```
[the remark this answers is not in the document]
Comment (pierventre) [reply app/app/src/test/java/org/omecproject/up4/impl/Up4TranslatorImplTest.java:46]: does it hurt ? I would say keep it here

**Review by pierventre — COMMENTED** [review]
minor

File: app/app/src/test/java/org/omecproject/up4/impl/Up4TranslatorImplTest.java (hunk 1 above)
[the remark this answers is not in the document]
Comment (daniele-moro) [reply app/app/src/test/java/org/omecproject/up4/impl/Up4TranslatorImplTest.java:46]: OK, let's keep it.

**Comment by pierventre** [comment]
there are ci failures - is that expected ?

**Review by pierventre — APPROVED** [review]
I had a deja-vu when I reviewed your last commit 🤣

**Pull request merged**